solidity/random-beacon/test/Utils.test.ts

```diff
@@ -0,0 +1,75 @@
+import { ethers } from "hardhat"
+import { expect } from "chai"
+import { hashDKGMembers } from "./utils/dkg"
+
+describe("utils/dkg", () => {
+  context("hashDKGMembers", () => {
```

Comment:
If we move the file as proposed in the previous comment we could reduce one level.

```suggestion
describe("hashDKGMembers", () => {
```

Reply:
2ca346e